src/field.rs

```diff
@@ -712,13 +712,42 @@ make_field!(
     8,
 );
 
-make_field!(
-    /// `GF(79228148845226978974766202881)`, a 96-bit field.
-    Field96,
-    u128,
-    FP96,
-    12,
-);
+mod field96 {
```

Comment:
Last suggestion: throw a doccomment on this module explaining its purpose so that future us won't inadvertently delete this otherwise useless module.